Classes/Queue/RabbitQueue.php

```diff
@@ -117,6 +117,21 @@ public function __construct(string $name, array $options = [])
         $queueOptions = $options['queueOptions'];
 
         $this->queueName = $queueOptions['name'] ?? $this->name;
+
+        $this->useDLX = (bool) ($options['useDLX'] ?? false);
+        if ($this->useDLX
+            && is_string($options['useDLX'])
```

Comment:
I'm actually not sure if we should mix string/bool values here. Wouldn't be another option like `dlxEchangeName` or so be more declarative?

Reply:
I though about that, too. But this would allow for `useDLX=false; dlxName=whatever` resulting in a setting that does nothing. So I'm equally unsatisfied with both.

Reply:
Yeah, but that would be intended by the user. By default there is no `dlxName` at all. We could and should only document it here: https://github.com/t3n/JobQueue.RabbitMQ/blob/master/Configuration/Settings.yaml#L26 

But by default there is not dlx setting at all. Personally i tend to a `dlxName` configuration